src/shivaniGDec19/assignment3/SumOfNumber.java

```diff
@@ -0,0 +1,22 @@
+package shivaniGDec19.assignment3;
+
+//WAP to find sum of user given number
+public class SumOfNumber {
+	static int digitSum(String input) {
+		int sum = 0;
+
+		for (int i = 0; i < input.length(); i++) {
+			System.out.println(input.charAt(i));
+			char currentChar = input.charAt(i);
+			String temp = String.valueOf(currentChar);
+			int x = Integer.parseInt(temp);
```

Comment:
int x = Integer.parseInt(String.valueOf(str.charAt(i)));
Hope you get this.. just efficient way I am describing.
Other things are good.